be/src/exec/pipeline/sink/export_sink_operator.cpp

```diff
@@ -81,40 +82,41 @@ void ExportSinkIOBuffer::close(RuntimeState* state) {
     SinkIOBuffer::close(state);
 }
 
-void ExportSinkIOBuffer::_process_chunk(bthread::TaskIterator<const ChunkPtr>& iter) {
+void ExportSinkIOBuffer::_process_chunk(bthread::TaskIterator<ChunkPtr>& iter) {
     --_num_pending_chunks;
     if (_is_finished) {
         return;
     }
```

Comment:
`--_num_pending_chunks;` should be updated after really done?

Reply:
> `--_num_pending_chunks;` should be updated after really done?

This variable is introduced to facilitate limiting the memory usage of sink operator by limiting the number of chunks pending to be processed. I think it's not a very strict limitation here, it's ok to be updated before or after it's really done.